### PR TITLE
Add myself as maintainer

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: Ben Ye <yb532204897@gmail.com>

* [ ] I added CHANGELOG entry for this change.
* [x] Change is not relevant to the end user.

## Changes

Change my role from triage to maintainer ^ ^

### CODE REVIEW

**Review by squat — APPROVED** [review]
:tada:

**Review by bwplotka — APPROVED** [review]

File: MAINTAINERS.md [hunk 1]
```diff
@@ -11,9 +11,10 @@
 | Prem Saraswat         | prmsrswt@gmail.com     | `@Prem Saraswat`         | [@prmsrswt](https://github.com/prmsrswt)    | Atlan             |
 | Marco Pracucci        | marco@pracucci.com     | `@pracucci`              | [@pracucci](https://github.com/pracucci)    | Grafana Labs      |
 | Matthias Loibl        | mail@matthiasloibl.com | `@metalmatze`            | [@metalmatze](https://github.com/metalmatze)| Red Hat           |
+| Ben Ye                | yb532204897@gmail.com  | `@yeya24`                | [@yeya24](https://github.com/yeya24)        | Red Hat           |
 
 We are bunch of people from different companies with various interests and skills.
-We are from different parts of Europe: Germany, Italy, Lithuania, Poland, and UK and India.
+We are from different parts of the world: Germany, Italy, Lithuania, Poland, UK, India and China.
```
Comment (bwplotka) [inline MAINTAINERS.md:17]: We are going truly Global!

**Review by kakkoyun — APPROVED** [review]
🎉

**Review by pracucci — APPROVED** [review]
Welcome and congrats! 🎉

**Review by prmsrswt — APPROVED** [review]
:tada:

**Pull request merged**